test/csuite/wt4105_large_doc_small_upd/main.c

```diff
@@ -132,7 +132,8 @@ main(int argc, char *argv[])
             modify_entry.data.size = strlen(modify_entry.data.data);
             modify_entry.offset = offset;
             modify_entry.size = modify_entry.data.size;
-            (void)alarm(1);
+            /* FIXME-PM-1521: extend timeout to pass the test */
+            (void)alarm(5);
```

Comment:
Do we understand why this is necessary now?

Reply:
I think we incur more overhead in lookaside eviction.